src/pages/gen2/build-a-backend/data/customize-authz/configure-custom-identity-and-group-claim/index.mdx

```diff
@@ -37,3 +37,24 @@ export type Schema = ClientSchema<typeof schema>;
 export const data = defineData({ schema });
 
 ```
+
+In your application, you can perform CRUD operations against the model using `client.models.<ModelName>` with the `userPool` auth mode.
+
+```ts
```

Comment:
can we add sample titles on these client-side examples to help convey this is client-side

Reply:
does something like `App.tsx` or `app/page.tsx` work? 👀

Reply:
I've been defaulting to `src/App.tsx` but yes I think either of those would work!

Reply:
Added file name titles!